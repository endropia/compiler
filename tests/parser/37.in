var a: Integer = 1;

begin
if a > 1 a := 2;
end.