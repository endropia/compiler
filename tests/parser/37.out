(4, 10) 'then' expected